server/schedule/operator.go

```diff
@@ -96,6 +102,7 @@ func (ap AddPeer) Influence(opInfluence OpInfluence, region *core.RegionInfo) {
 
 	to.RegionSize += region.GetApproximateSize()
 	to.RegionCount++
+	to.StepCost += RegionWeight
```

Comment:
In theory, a cost should be added to the leader of the region. Maybe add a TODO here.